packages/headless/doc-parser/doc-parser.ts

```diff
@@ -13,6 +13,8 @@ import {
   resolveEngine,
 } from './src/headless-export-resolvers/engine-resolver';
 import {caseAssistUseCase} from './use-cases/case-assist';
+// TODO: Uncomment when we're ready to make the Commerce sub-package public.
```

Comment:
We can link this to CAPI-89. Same for the todo below!
```suggestion
// TODO CAPI-89: Uncomment when we're ready to make the Commerce sub-package public.
```